### PR TITLE
Miller & Bregman 2015

### PR DESCRIPTION
**Pull request by profxj**
Adds in their Halo profile.

docs and tests and Notebook too.

### CODE REVIEW

File: docs/nb/Galactic_Halo.ipynb [hunk 1]
```diff
@@ -0,0 +1,532 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "Rperp = 8. * units.kpc # e.g. Sun\n",
+    "rmax = 1.\n",
```
Comment (yzhenggit) [inline docs/nb/Galactic_Halo.ipynb]: add that rmax is in unit of rvir?

File: frb/halos.py [hunk 2]
```diff
@@ -882,6 +882,40 @@ def rho_b(self, xyz):
...
+
+
+    """
+    def __init__(self, log_Mhalo=12.18, c=7.67, f_hot=0.75, **kwargs):
+        # Init ModifiedNFW
+        ModifiedNFW.__init__(self, log_Mhalo=log_Mhalo, c=c, f_hot=f_hot, **kwargs)
+
+        # Best parameters
+        self.beta = 0.45
+        self.n0_rc3b = 0.79e-2  # Last entry of Table 2; Crazy units
```
Comment (yzhenggit) [inline frb/halos.py]: Why choosing these two best fits of OVII? From their Table 2, second line, it seems like if use OVIII parameters, then n0_rc3b would be roughly twice as much? (p.s., I didn't read their paper throughout to understand their model... maybe you can add a line of comment here about the choice of parameters? )

File: frb/halos.py [hunk 3]
```diff
@@ -882,6 +882,40 @@ def rho_b(self, xyz):
...
+              Coordinates in kpc
+
+        Returns:
+            ndarray: Number density with units of 1/cm**3
+
+        """
+        radius = np.sqrt(rad3d2(xyz))
+        #
+        #nH = self.n0 * (1+ (radius.self.r_c)**2)**(-3*self.beta/2)
+        nH = self.n0_rc3b / radius**(3*self.beta)
```
Comment (yzhenggit) [inline frb/halos.py]: how about adding a comment say this is from Eq (2) of their paper?

Comment (yzhenggit) [reply frb/halos.py]: and, where is nHe in this equation? should nH be some factor smaller?

File: docs/nb/Galactic_Halo.ipynb (hunk 1 above)
Comment (profxj) [reply docs/nb/Galactic_Halo.ipynb]: good suggestion. done

File: frb/halos.py (hunk 2 above)
Comment (profxj) [reply frb/halos.py]: Actually, it looks like the mass doesn't vary so much in their 
range of models.  Adding a note.

File: frb/halos.py (hunk 3 above)
Comment (profxj) [reply frb/halos.py]: done.

n_e is a method in the parent class:  `ModifiedNFW`

**Comment by yzhenggit** [comment]
all set!

**Comment by profxj** [comment]
merging

**Pull request merged**